nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -190,8 +190,10 @@ def setup_experiment(self, config):
         # Configure distribute pytorch
         self.distributed = config.get("distributed", False)
         self.rank = config.get("rank", 0)
+        tcp_port = config.get("tcp_port", 54321)
+
         if self.distributed:
-            dist_url = config.get("dist_url", "tcp://127.0.0.1:54321")
+            dist_url = config.get("dist_url", f"tcp://127.0.0.1:{tcp_port}")
```

Comment:
The caller should update the `dist_url` parameter

Reply:
Fixed